.ci/manywheel/build_common.sh

```diff
@@ -111,12 +111,6 @@ case ${DESIRED_PYTHON} in
     ;;
 esac
 
-if [[ "$DESIRED_DEVTOOLSET" == *"cxx11-abi"* ]]; then
```

Comment:
This conditional in particular was problematic and was causing particular builds (like the full builds + s390x builds) to fail out and build incorrectly for pre-cxx11 abi.


See example failures:

* https://github.com/pytorch/pytorch/actions/runs/13917778985/job/38951860549
* https://github.com/pytorch/pytorch/actions/runs/13917778968/job/38953352361